wasp/src/main/java/com/orhanobut/wasp/OkHttpStack.java

```diff
@@ -0,0 +1,35 @@
+package com.orhanobut.wasp;
+
+import com.android.volley.toolbox.HurlStack;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.OkUrlFactory;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * @author Emmar Kardeslik
+ */
+public class OkHttpStack extends HurlStack {
+
+    private final OkUrlFactory factory;
+
+    public OkHttpStack() {
+        this(new OkHttpClient());
+    }
+
+    public OkHttpStack(final OkHttpClient client) {
+        if (client == null) {
+            throw new NullPointerException("Client may not be null.");
```

Comment:
I would say HttpClient may not be null to minimize the confusing, Client may sound a little confusing